main.rb

```diff
@@ -1,12 +1,36 @@
 require_relative 'app'
+require 'json'
 
-# Main
-def main
-  app = App.new
-  loop do
-    app.run
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  def actions
     puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
```

Comment:
- [ ] Kindly ensure the book details are displayed. This is a refactor for data storage so the previous functionality is still expected to be there.

  ![djdye](https://user-images.githubusercontent.com/81468675/211836643-d1952c40-79e3-406b-ad27-714abe409eb4.jpg)